helm/files/logs.conf

```diff
@@ -0,0 +1,2 @@
+LogFormat "{ \"time\":\"%{%Y-%m-%dT%T}t.%{usec_frac}t%{%z}t\", \"remoteIP\":\"%a\", \"host\":\"%V\", \"requestPath\":\"%U\", \"query\":\"%q\", \"method\":\"%m\", \"status\":\"%>s\", \"userAgent\":\"%{User-agent}i\", \"referer\":\"%{Referer}i\", \"timetaken\":\"%D\"  }" json
+ErrorLogFormat "{ \"time\":\"%{cu}tZ\", \"severity\" : \"%l\", \"message\" : \"%M\"}"
```

Comment:
Is this what eventually becomes `/etc/apache2/sites-available/logs.conf`? if so the format looks a bit different from the usual apache log format in other projects

Reply:
Yeah this is also leftover stuff, I have removed this now as we now have this in the new base containers.